internal/build/backward_test.go

```diff
@@ -1,88 +0,0 @@
-package build
-
-import (
```

Comment:
not related to this pr, revert please

Reply:
related, as I sent you: an import cycle. It shouldn't be here.